server/views/pages/approve/cases.test.ts

```diff
@@ -32,4 +33,19 @@ describe('View and print a licence - case list', () => {
     expect($('#nomis-id-2').text()).toBe('A1234AB')
     expect($('#release-date-2').text()).toBe('1 Sep 2022')
   })
+
+  it('should display HDC release warning label when kind is HDC', () => {
```

Comment:
could add another test for approval needed : false to show it not shown

Reply:
Added another test to cover highlighting HDC cases in both approval needed and recently approved tab views.